devstack/lib/nfp

```diff
@@ -178,54 +159,16 @@ function create_nfp_image {
-
-            VisibilityQcow2ImageName=visibility
-            VisibilityInstanceName="VisibilityVM_instance"
-            create_port_for_vm $VisibilityQcow2ImageName
-            if [[ $VisibilityQcow2Image = build ]]; then
-                # edits the docker file to add visibility vm IP address
-                configure_vis_ip_addr_in_docker
-                # prepare visibility source, this is needed for diskimage build
-                cd /home/stack/
-                sudo rm -rf visibility
-                sudo git clone https://$GIT_ACCESS_USERNAME:$GIT_ACCESS_PASSWORD@github.com/oneconvergence/visibility.git -b $VISIBILITY_GIT_BRANCH
-                echo "Building Image: $VisibilityQcow2ImageName"
-                sudo python $DISKIMAGE_CREATE_DIR/visibility_disk_image_create.py $DISKIMAGE_CREATE_DIR/visibility_conf.json $GBPSERVICE_BRANCH $DEVSTACK_DIR/local.conf
-                VisibilityQcow2Image=$(cat /tmp/image_path)
-            fi
-            echo "Uploading Image: $VisibilityQcow2ImageName"
-            glance image-create --name $VisibilityQcow2ImageName --disk-format qcow2 --container-format bare --visibility public --file $VisibilityQcow2Image
-
-            AsavQcow2ImageName=asav
-            echo "Uploading Image: $AsavQcow2ImageName"
-            glance image-create --name $AsavQcow2ImageName --disk-format qcow2 --container-format bare --visibility public --file $AsavQcow2Image
-
-            PaloAltoQcow2ImageName=paloalto
-            echo "Uploading Image: $PaloAltoQcow2ImageName"
-            glance image-create --name $PaloAltoQcow2ImageName --disk-format qcow2 --container-format bare --visibility public --file $PaloAltoQcow2Image
-        else
-            ConfiguratorQcow2ImageName=configurator
-            ConfiguratorInstanceName="configuratorVM_instance"
-            create_port_for_vm $ConfiguratorQcow2ImageName
-            if [[ $ConfiguratorQcow2Image = build ]]; then
-                echo "Building Image: $ConfiguratorQcow2ImageName"
-                sudo python $DISKIMAGE_CREATE_DIR/disk_image_create.py $DISKIMAGE_CREATE_DIR/configurator_conf.json $GBPSERVICE_BRANCH
-                ConfiguratorQcow2Image=$(cat /tmp/image_path)
-            fi
-            echo "Uploading Image: $ConfiguratorQcow2ImageName"
-            glance image-create --name $ConfiguratorQcow2ImageName --disk-format qcow2 --container-format bare --visibility public --file $ConfiguratorQcow2Image
+        ConfiguratorQcow2ImageName=configurator
+        ConfiguratorInstanceName="configuratorVM_instance"
+        create_port_for_vm $ConfiguratorQcow2ImageName $ConfiguratorInstanceName
+        if [[ $ConfiguratorQcow2Image = build ]]; then
+            echo "Building Image: $ConfiguratorQcow2ImageName"
+            sudo python $DISKIMAGE_CREATE_DIR/disk_image_create.py $DISKIMAGE_CREATE_DIR/configurator_conf.json $GBPSERVICE_BRANCH
+            ConfiguratorQcow2Image=$(cat /tmp/image_path)
```

Comment:
Can we create this in disk image create dir. There is a chance that it will get cleaned up by the time we use it.